src/media/css/account.styl

```diff
@@ -81,6 +81,10 @@
         display: none;
     }
 }
+.html-rtl .region span {
```

Comment:
This is closer to the AMO RTL strategy where we follow each block with `.html-rtl {...}` but I am not fanatic about the organization. If we agree on separate files I'll move them.